tango/executors/multicore_executor.py

```diff
@@ -217,7 +218,7 @@ def _try_to_execute_next_step(config_path: str, run_name: Optional[str] = None)
             if len(_queued_steps) == 0:
                 logger.debug("No steps in queue!")
                 return
-            if len(_running) < self.parallelism:
+            if len(_running) < (self.parallelism or 1):
```

Comment:
Why the extra check with `or 1` if we've already updated `self.parallelism` on line 36?

Reply:
Just for safety since `self.parallelism` is technically a public attribute, and so can be easily modified in-place.